modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/CreationMenu.tsx

```diff
@@ -86,7 +87,13 @@ const DropDown = ({
 								loadData,
 							});
 
-							if (item.href || item.target) {
+							if (onCreationActionClick) {
+								onCreationActionClick({
+									action: item?.data?.action,
```

Comment:
Should we expose the whole `item` here? I would expect devs would need other properties of action, like `href`, to build custom logic upon it.